tests/common.bash

```diff
@@ -819,6 +819,7 @@ function arch_to_golang() {
 
 	case "${arch}" in
 		aarch64) echo "arm64";;
+		arm64) echo "${arch}";;
```

Comment:
I'd have added `aarch64|arm64) echo "arm64" ;; ` as I think that's easier to read at first glance.

Reply:
Updated - thanks!